modules/PublicLockV14EligibilityHatter.json

```diff
@@ -0,0 +1,142 @@
+{
+  "name": "Unlock Protocol Eligibility (PublicLock V14)",
```

Comment:
Yes, the consumer of this information is user of an app like the Hats app.

I like this suggestion, but want to confirm that you're comfortable with abstracting away the public lock/unlock stuff. Even if so, I do think we should include a reference to Unlock so that users know / can view or manage the lock configuration from your app